testing/test_tmpdir.py

```diff
@@ -25,12 +26,29 @@ def test_ensuretemp(recwarn):
     assert d1.check(dir=1)
 
 
+@attr.s
+class FakeConfig(object):
```

Comment:
TBH not sure if this fake actually improves things... the "option returns self" and `get` are quite the hack. 😬

Reply:
its indeed quite a hack, we cant make a sane config object for use in such cases